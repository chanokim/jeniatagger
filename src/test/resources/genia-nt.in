Hi

This is a test for genia -nt ( tokenization given ) == (x) is not the same as ( x )

CD4-CD8
CAPITALED
mizpeled
DEUTSCH

word
words
Word
Words
wOrD
wOrDs
wOrDS
Multiple words is a complex wording using words for mere word - pleasure .

@.@

p53

The p53 protein

in

while

killing

killing machine

He is in trouble

red

Sun

He is blind

He is American

He is Robert and American and blind and very much in trouble .

?
.
-

mind the gap (the following tests contain import whitespaces for the tests. Make sure they're not automatically trimmed!)



 leading space

trailing space

big        space




CD4-CD8 alpha protein binds with TH - 8 receptor

bla
bla
bla

!!!!!

23472ejf828&*(&(&#34j29ejf92&(*&(&#$

53

_

:
8
;
`

(btw) ( btw ) Blank lines are not parsed and left as a blank line .








:-)



----------------------------------------------------------------------
