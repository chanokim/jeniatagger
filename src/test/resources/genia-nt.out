Hi	Hi	NNP	B-NP	O


This	This	DT	B-NP	O
is	be	VBZ	B-VP	O
a	a	DT	B-NP	O
test	test	NN	I-NP	O
for	for	IN	B-PP	O
genia	genia	NN	B-NP	O
-nt	-nt	NN	I-NP	O
(	(	(	O	O
tokenization	tokenization	NN	B-NP	O
given	give	VBN	B-VP	O
)	)	)	O	O
==	==	SYM	B-NP	O
(x)	(x)	LS	B-LST	O
is	be	VBZ	B-VP	O
not	not	RB	O	O
the	the	DT	B-NP	O
same	same	JJ	I-NP	O
as	as	IN	B-PP	O
(	(	(	O	O
x	x	NN	B-NP	O
)	)	)	O	O


CD4-CD8	CD4-CD8	NN	B-NP	O

CAPITALED	CAPITALED	VBN	B-VP	O

mizpeled	mizpele	VBN	B-VP	O

DEUTSCH	DEUTSCH	NN	B-NP	O


word	word	NN	B-NP	O

words	word	NNS	B-NP	O

Word	Word	NNP	B-NP	O

Words	Word	NNS	B-NP	O

wOrD	wOrD	NN	B-NP	O

wOrDs	wOrD	NNS	B-NP	O

wOrDS	wOrDS	NN	B-NP	O

Multiple	Multiple	JJ	B-NP	O
words	word	NNS	I-NP	O
is	be	VBZ	B-VP	O
a	a	DT	B-NP	O
complex	complex	JJ	I-NP	O
wording	wording	NN	I-NP	O
using	use	VBG	B-VP	O
words	word	NNS	B-NP	O
for	for	IN	B-PP	O
mere	mere	JJ	B-NP	O
word	word	NN	I-NP	O
-	-	HYPH	B-NP	O
pleasure	pleasure	NN	I-NP	O
.	.	.	O	O


@.@	@.@	LS	B-LST	O


p53	p53	NN	B-NP	B-protein


The	The	DT	B-NP	O
p53	p53	NN	I-NP	B-protein
protein	protein	NN	I-NP	I-protein


in	in	IN	B-PP	O


while	while	IN	B-SBAR	O


killing	kill	VBG	B-VP	O


killing	kill	VBG	B-VP	O
machine	machine	NN	B-NP	O


He	He	PRP	B-NP	O
is	be	VBZ	B-VP	O
in	in	IN	B-PP	O
trouble	trouble	NN	B-NP	O


red	red	JJ	O	O


Sun	Sun	NNP	B-NP	O


He	He	PRP	B-NP	O
is	be	VBZ	B-VP	O
blind	blind	JJ	B-ADJP	O


He	He	PRP	B-NP	O
is	be	VBZ	B-VP	O
American	American	JJ	B-ADJP	O


He	He	PRP	B-NP	O
is	be	VBZ	B-VP	O
Robert	Robert	NNP	B-NP	O
and	and	CC	O	O
American	American	JJ	B-NP	O
and	and	CC	I-NP	O
blind	blind	JJ	I-NP	O
and	and	CC	O	O
very	very	RB	B-ADVP	O
much	much	RB	I-ADVP	O
in	in	IN	B-PP	O
trouble	trouble	NN	B-NP	O
.	.	.	O	O


?	?	.	O	O

.	.	.	O	O

-	-	HYPH	O	O


mind	mind	VB	B-VP	O
the	the	DT	B-NP	O
gap	gap	NN	I-NP	O
(the	(the	DT	B-NP	O
following	follow	VBG	I-NP	O
tests	test	NNS	I-NP	O
contain	contain	VBP	B-VP	O
import	import	NN	B-NP	O
whitespaces	whitespace	NNS	I-NP	O
for	for	IN	B-PP	O
the	the	DT	B-NP	O
tests.	tests.	NN	I-NP	O
Make	Make	NN	I-NP	O
sure	sure	JJ	I-NP	O
they're	they're	NN	I-NP	O
not	not	RB	O	O
automatically	automatically	RB	B-ADVP	O
trimmed!)	trimmed!)	NN	B-NP	O




leading	lead	VBG	B-NP	O
space	space	NN	I-NP	O


trailing	trail	VBG	B-VP	O
space	space	NN	B-NP	O


big	big	JJ	B-NP	O
space	space	NN	I-NP	O





CD4-CD8	CD4-CD8	JJ	B-NP	B-protein
alpha	alpha	NN	I-NP	I-protein
protein	protein	NN	I-NP	I-protein
binds	bind	VBZ	B-VP	O
with	with	IN	B-PP	O
TH	TH	NN	B-NP	B-protein
-	-	HYPH	O	I-protein
8	8	CD	B-NP	I-protein
receptor	receptor	NN	I-NP	I-protein


bla	bla	LS	B-LST	O

bla	bla	LS	B-LST	O

bla	bla	LS	B-LST	O


!!!!!	!!!!!	.	O	O


23472ejf828&*(&(&#34j29ejf92&(*&(&#$	23472ejf828&*(&(&#34j29ejf92&(*&(&#$	$	O	O


53	53	CD	B-NP	O


_	_	LS	B-LST	O


:	:	:	O	O

8	8	CD	B-NP	O

;	;	:	O	O

`	`	``	O	O


(btw)	(btw)	LS	B-LST	B-cell_line
(	(	(	O	I-cell_line
btw	btw	NN	B-NP	I-cell_line
)	)	)	O	I-cell_line
Blank	Blank	NNP	B-NP	I-cell_line
lines	line	NNS	I-NP	I-cell_line
are	be	VBP	B-VP	O
not	not	RB	I-VP	O
parsed	parse	VBN	I-VP	O
and	and	CC	I-VP	O
left	leave	VBN	I-VP	O
as	as	IN	B-PP	O
a	a	DT	B-NP	O
blank	blank	JJ	I-NP	O
line	line	NN	I-NP	O
.	.	.	O	O









:-)	:-)	LS	B-LST	O




----------------------------------------------------------------------	----------------------------------------------------------------------	SYM	O	O

